src/pages/LettersOrgs.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactElement } from "react";
+import LogoRow from "src/components/LogoRow/LogoRow";
+import Hero from "src/components/sections/Orgs/Hero";
+import { PartnerOrgs } from "src/utils/constants";
+import UseCases from "src/components/sections/Orgs/UseCases";
+import OrgsTestimonials from "src/components/sections/Orgs/OrgTestimonials";
+
+export default function LettersOrgs(): ReactElement {
+  return (
+    <div>
+      <Hero />
+      {/* <LogoRow images={PartnerOrgs} title="Trusted by" size="logo-row-large"/> */}
```

Comment:
?

Reply:
we havent gotten permission yet to use their logos :Xxxxx

Reply:
i'll uncomment as soon as we have permission lol